www/js/util.js

```diff
@@ -1433,6 +1435,13 @@ function sendVideoUpdate() {
 
 /* chat */
 
+function stripImages(msg){
+    if (!USEROPTS.strip_image) {
+        return msg;
+    }
+    return msg.indexOf('<img') > -1 ? msg.replace(IMAGE_MATCH, "$1") : msg;
```

Comment:
I think that would double up the linkage for most channels, since this only captures the img, and the img is usually already wrapped in an anchor in most channels img embed codes.